doc/tutorials.rst

```diff
@@ -50,6 +50,11 @@ algorithms using PennyLane and near-term quantum hardware.
     :figure: ../examples/figures/NOON.png
     :description: :ref:`state_preparation`
 
+.. customgalleryitem::
+    :tooltip: Ising model example with PennyLane PyTorch interface.
+    :figure: ../examples/figures/pytorch.png
```

Comment:
Please don't forget to remove any no-longer-needed images from git (e.g. tensorflow logo)